magsnake.md

```diff
@@ -0,0 +1,125 @@
+---
+name: "@magsnake"
+project: "  Multi-purpose Smart Robot"
+---
+
+# Project Name
+
+  Multi-purpose Smart Robot
+
+## Summary
+
+  I am going to build a mini robot which can solve questions, make decisions, manage digital activities and serve as a virtual assistant.
```

Comment:
what specific features will your mini robot have? and, how do you plan on interacting with this robot?

Reply:
It can detect objects, search images, manage app time for other devices, serve as AI Assistant ad can tell climatic conditions.
It can be used as a pass-key for my accounts.
It could manage networks.
And I will make it to do a lot more things.